dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/MetadataDumperTest.java

```diff
@@ -175,7 +173,7 @@ public void testFailsOnUnrecognizedFlagForSpecificDialect() {
 
     // Assert
     assertEquals(
-        "Property: name='hiveql.rpc.protection', value='privacy' is not compatible with connector"
+        "Property: name='Dhiveql.rpc.protection', value='privacy' is not compatible with connector"
```

Comment:
I would keep the name as `hiveql.rpc.protection`, because `Dhiveql.rpc.protection` may be confusing (unless prefixed with `-D`).